src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java

```diff
@@ -149,4 +144,15 @@ public String toString() {
                 .add("detect-connected-devices", detectConnectedDevices)
```

Comment:
please add inner vlan into toString() and into hashCode()

Reply:
done